### ISSUE TITLE
Unable to use plugin with git@github.com:owner/repo.git repository URL format

### ISSUE THREAD

**Issue by vgrigoruk** [mikeknox/aws-paramstore-secrets-buildkite-plugin#8]
global environment hook fails with 
```
Traceback (most recent call last):
--
  | File "/usr/local/bin/bk-ssm-secrets", line 11, in <module>
  | load_entry_point('bk-ssm-secrets==0.5.0', 'console_scripts', 'bk-ssm-secrets')()
  | File "/usr/local/lib/python3.7/site-packages/bk_ssm_secrets-0.5.0-py3.7.egg/bk_ssm_secrets/__main__.py", line 14, in main
  | File "/usr/local/lib/python3.7/site-packages/bk_ssm_secrets-0.5.0-py3.7.egg/bk_ssm_secrets/helpers.py", line 66, in url_to_slug
  | ValueError: Invalid URL scheme found: git@github.com:<orgname>/<reponame>.git
  | 🚨 Error: Error setting up bootstrap: The global environment hook exited with status 1
```

**Comment by xiaket** [comment]
Thanks @vgrigoruk for reporting this, we will take a look today and get back to you.

**Comment by xiaket** [comment]
Hi @vgrigoruk! Thanks for your interest in this repo and our effort to secure Buildkite.

The issue we are seeing here is caused by a piece of code that checks the URL scheme(usually either `ssh://` or `https://`, but it looks like Buildkite is happy without this part. So it makes no sense for us to check it here. I've created a PR to remove this.

Also, if you've got some time, you may (or may not) find this blog article interesting, as is described in detail our setup and why it looks like this today: https://blog.xiaket.org/2020/securing-buildkite.html

**Comment by vgrigoruk** [comment]
Hi @xiaket. Thank you for looking into the issue.
I've tried to test the method without a check for url scheme and it produces unexpected results for scp-style urls like `git@github.com:owner/repo.git` (it returns `None_git@github.com:a_b.git` instead of `github.com_a_b.git`
I'll create  a PR that fixes `url_to_slug` method, so it handles both types of URLs correctly.

**Comment by mikeknox** [comment]
thanks @vgrigoruk ,
I've merged @xiaket PR to remove the schema check.

**Comment by mikeknox** [comment]
also merged #10 in as v0.7.4
